lib/widgets/autocomplete.dart

```diff
@@ -223,28 +229,62 @@ class ComposeAutocomplete extends AutocompleteField<ComposeAutocompleteQuery, Co
 }
 
 class _MentionAutocompleteItem extends StatelessWidget {
-  const _MentionAutocompleteItem({required this.option});
+  const _MentionAutocompleteItem({required this.option, required this.narrow});
 
   final MentionAutocompleteResult option;
+  final Narrow narrow;
 
   @override
   Widget build(BuildContext context) {
+    final store = PerAccountStoreWidget.of(context);
     Widget avatar;
-    String label;
+    Widget label;
     switch (option) {
       case UserMentionAutocompleteResult(:var userId):
-        avatar = Avatar(userId: userId, size: 32, borderRadius: 3);
-        label = PerAccountStoreWidget.of(context).users[userId]!.fullName;
+        avatar = Avatar(userId: userId, size: 32, borderRadius: 3); // web uses 21px
+        label = Text(store.users[userId]!.fullName);
+      case WildcardMentionAutocompleteResult(:var wildcard):
+        avatar = const Icon(ZulipIcons.three_person, size: 29); // web uses 19px
+        label = wildcardLabel(wildcard, context: context, store: store);
     }
 
     return Padding(
       padding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 8.0),
       child: Row(children: [
         avatar,
         const SizedBox(width: 8),
-        Text(label),
+        label,
       ]));
   }
+
+  Widget wildcardLabel(Wildcard wildcard, {
+    required BuildContext context,
+    required PerAccountStore store,
+  }) {
+    final isDmNarrow = narrow is DmNarrow;
+    final isChannelWildcardAvailable = store.account.zulipFeatureLevel >= 247; // TODO(server-9)
+    final localizations = ZulipLocalizations.of(context);
+    final description = switch (wildcard) {
+      Wildcard.all => isDmNarrow
+        ? localizations.notifyRecipients
+        : isChannelWildcardAvailable
+            ? localizations.notifyChannel
+            : localizations.notifyStream,
+      Wildcard.everyone => isDmNarrow
+        ? localizations.notifyRecipients
+        : isChannelWildcardAvailable
+            ? localizations.notifyChannel
+            : localizations.notifyStream,
+      Wildcard.channel => localizations.notifyChannel,
+      Wildcard.stream => isChannelWildcardAvailable
+        ? localizations.notifyChannel
+        : localizations.notifyStream,
+      Wildcard.topic => localizations.notifyTopic,
+    };
+    return Text.rich(TextSpan(text: '${wildcard.name} ', children: [
+      TextSpan(text: description, style: TextStyle(fontSize: 12,
+        color: Colors.black.withValues(alpha: 0.8)))]));
```

Comment:
Web uses 85% of the name (i.e., **all**) font size for the description (i.e., **Notify channel**). For us, 85% or 14 is almost 12.